src/grid.c

```diff
@@ -1135,8 +1135,7 @@ SEXP L_convert(SEXP x, SEXP whatfrom,
          * In these cases do NOT transform thru INCHES 
          * (to avoid divide-by-zero, but still do something useful)
          */
-        relConvert = (!isUnitArithmetic(x) && !isUnitList(x) &&
-                      (unitUnit(x, i) == L_NATIVE || unitUnit(x, i) == L_NPC) &&
+        relConvert = ((unitUnit(x, i) == L_NATIVE || unitUnit(x, i) == L_NPC) &&
```

Comment:
Checking my thinking:  this is ok because unitArithmetic and unitList are now just different types of unit, so the (unitUnit(x, i) == L_NATIVE || unitUnit(x, i) == L_NPC) automatically excludes unitArithmetic and unitList (?)

Reply:
true — for unitArithmetic at least, there is no special jnitList anymore

Reply:
true — for unitArithmetic at least, there is no special jnitList anymore